### DISCUSSION TITLE
Folder

### DISCUSSION THREAD

**Discussion by ChurroC** [zen-browser/desktop#2618] [Ideas]
Just like Arc it would be great to store websites into folder or tab groups. Even though it isn't naturally in Firefox would it be possible to add it an extra feature in Zen.

**Comment by sm-riyadh** [comment]
Is it possible to make it happen using the addons? I can give it a try given its just JS

**Comment by Lu9-ST** [comment]
if I recall that exact feature is being worked on *as we speak* - in the meantime, there is an extension called Sidebery, with a matching Zen version and Zen theme/mod to further integrate it into the Zen Browser workflow.
